e2e/images_test.go

```diff
@@ -76,7 +76,7 @@ Deleted: b-simple-app:latest`,
 	})
 }
 
-func TestImageTag(t *testing.T) {
+func TestImmakeageTag(t *testing.T) {
```

Comment:
`TestImmakeageTag` ?

Reply:
oh, nice confusion between IDE and terminal :P